packages/language/src/document.js

```diff
@@ -0,0 +1,741 @@
+}
+
+/**
+ * The parts of speech that will be recognized by the Natural Language API.
+ *
+ * @private
+ * @type {object}
+ */
+Document.PART_OF_SPEECH = {
+  UNKNOWN: 'Unknown',
+  ADJ: 'Adjective',
+  ADP: 'Adposition (preposition and postposition)',
+  ADV: 'Adverb',
+  CONJ: 'Conjunction',
+  DET: 'Determiner',
+  NOUN: 'Noun (common and proper)',
+  NUM: 'Cardinal number',
+  PRON: 'Pronoun',
+  PRT: 'Particle or other function word',
+  PUNCT: 'Punctuation',
+  VERB: 'Verb (all tenses and modes)',
+  X: 'Other: foreign words, typos, abbreviations',
+  AFFIX: 'Affix'
+};
+
+/**
+ * Run an annotation of the text from the document.
+ *
+ * By default, all annotation types are requested:
+ *
+ *   - The sentiment of the document (positive or negative)
+ *   - The entities of the document (people, places, things, etc.)
+ *   - The syntax of the document (adjectives, nouns, verbs, etc.)
+ *
+ * See the examples below for how to request a subset of those types.
+ *
+ * If you only need one type of annotation, you may appreciate one of these
+ * other methods from our API:
+ *
+ *   - {module:language#detectEntities}
+ *   - {module:language#detectSentiment}
+ *
+ * @resource [documents.annotateText API Documentation]{@link https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText}
+ *
+ * @param {object=} options - Configuration object.
+ * @param {boolean} options.entities - Detect the entities from this document.
+ *     By default, all features (`entities`, `sentiment`, and `syntax`) are
+ *     enabled. By overriding any of these values, all defaults are switched to
+ *     `false`.
+ * @param {boolean} options.sentiment - Detect the sentiment from this document.
+ *     By default, all features (`entities`, `sentiment`, and `syntax`) are
+ *     enabled. By overriding any of these values, all defaults are switched to
+ *     `false`.
+ * @param {boolean} options.syntax - Detect the syntax from this document. By
+ *     default, all features (`entities`, `sentiment`, and `syntax`) are
+ *     enabled. By overriding any of these values, all defaults are switched to
+ *     `false`.
+ * @param {boolean} options.verbose - Enable verbose mode for more detailed
+ *     results. Default: `false`
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error occurred while making this request.
+ * @param {object} callback.annotation - The formatted API response.
+ * @param {string} callback.annotation.language - The language detected from the
+ *     text.
+ * @param {number} callback.annotation.sentiment - A value in the range of
+ *     `-100` to `100`. Large numbers represent more postiive sentiments.
+ * @param {object} callback.annotation.entities - The recognized entities from
+ *     the text, grouped by the type of entity.
+ * @param {string[]} callback.annotation.entities.art - Art entities detected
+ *     from the text. This is only present if detections of this type were
```

Comment:
Do you think it would be better if we made a note about keys missing within the `callback.annotation.entities` description instead of repeating the text?